File: OPENAPI-REST-API/swagger-client/java-vertx/src/main/java/io/swagger/server/api/model/DnsNewDomain.java
package io.swagger.server.api.model;

import java.util.Objects;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonCreator;
import io.swagger.v3.oas.annotations.media.Schema;
import io.vertx.core.json.JsonObject;
import io.vertx.codegen.annotations.DataObject;
/**
 * The request for a new domain to be managed by the dns servers.
 */




public class DnsNewDomain   {
  private String domain = null;

  private String ip = null;

  public DnsNewDomain domain(String domain) {
    this.domain = domain;
    return this;
  }

  /**
   * The domain name.
   * @return domain
   **/
    public String getDomain() {
    return domain;
  }

  public void setDomain(String domain) {
    this.domain = domain;
  }

  public DnsNewDomain ip(String ip) {
    this.ip = ip;
    return this;
  }

  /**
   * IP Address to point the domain to.
   * @return ip
   **/
    public String getIp() {
    return ip;
  }

  public void setIp(String ip) {
    this.ip = ip;
  }


  @Override
  public boolean equals(java.lang.Object o) {
    if (this == o) {
      return true;
    }
    if (o == null || getClass() != o.getClass()) {
      return false;
    }
    DnsNewDomain dnsNewDomain = (DnsNewDomain) o;
    return Objects.equals(this.domain, dnsNewDomain.domain) &&
        Objects.equals(this.ip, dnsNewDomain.ip);
  }

  @Override
  public int hashCode() {
    return Objects.hash(domain, ip);
  }

  @Override
  public String toString() {
    StringBuilder sb = new StringBuilder();
    sb.append("class DnsNewDomain {\n");
    
    sb.append("    domain: ").append(toIndentedString(domain)).append("\n");
    sb.append("    ip: ").append(toIndentedString(ip)).append("\n");
    sb.append("}");
    return sb.toString();
  }

  /**
   * Convert the given object to string with each line indented by 4 spaces
   * (except the first line).
   */
  private String toIndentedString(java.lang.Object o) {
    if (o == null) {
      return "null";
    }
    return o.toString().replace("\n", "\n    ");
  }
}

File: OPENAPI-REST-API/swagger-client/dart/lib/model/dns_new_domain.dart
part of swagger.api;

class DnsNewDomain {
  /* The domain name. */
  String domain = null;
/* IP Address to point the domain to. */
  String ip = null;

  DnsNewDomain();

  @override
  String toString() {
    return 'DnsNewDomain[domain=$domain, ip=$ip, ]';
  }

  DnsNewDomain.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    domain = json['domain'];
    ip = json['ip'];
  }

  Map<String, dynamic> toJson() {
    return {
      'domain': domain,
      'ip': ip
     };
  }

  static List<DnsNewDomain> listFromJson(List<dynamic> json) {
    return json == null ? new List<DnsNewDomain>() : json.map((value) => new DnsNewDomain.fromJson(value)).toList();
  }

  static Map<String, DnsNewDomain> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, DnsNewDomain>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new DnsNewDomain.fromJson(value));
    }
    return map;
  }
}
